### PR TITLE
meeting: use quicktest

### PR DESCRIPTION
**Pull request by rogpeppe**
Step 1 of quite a few.

### CODE REVIEW

File: meeting/meeting_test.go [hunk 1]
```diff
@@ -49,38 +35,40 @@ type nilMetrics struct{}
 func (nilMetrics) RequestCompleted(startTime time.Time) {}
 func (nilMetrics) RequestsExpired(count int)            {}
 
-func (s *suite) TestRendezvousWaitBeforeDone(c *gc.C) {
-	s.PatchValue(&meeting.Clock, s.clock)
+func TestRendezvousWaitBeforeDone(t *testing.T) {
+	c := qt.New(t)
```
Comment (mhilton) [inline meeting/meeting_test.go]: defer c.Done() otherwise your patch will bleed into other tests.

File: meeting/meeting_test.go [hunk 2]
```diff
@@ -49,38 +35,40 @@ type nilMetrics struct{}
...
 	count := int32(0)
-	store := newFakeStore(&count, s.clock)
+	store := newFakeStore(&count, clock)
 	m, err := meeting.NewPlace(meeting.Params{
 		Store:      store,
 		ListenAddr: "localhost",
 		DisableGC:  true,
 	})
-	c.Assert(err, gc.IsNil)
+	c.Assert(err, qt.IsNil)
```
Comment (mhilton) [inline meeting/meeting_test.go]: if you're changing things make it c.Assert(err, qt.Equals, nil)

File: meeting/meeting_test.go [hunk 3]
```diff
@@ -89,99 +77,103 @@ func (s *suite) TestRendezvousWaitBeforeDone(c *gc.C) {
...
+	c.Assert(err, qt.ErrorMatches, `rendezvous ".*" not found`)
 
-	c.Assert(atomic.LoadInt32(&count), gc.Equals, int32(0))
+	c.Assert(atomic.LoadInt32(&count), qt.Equals, int32(0))
 }
 
-func (s *suite) TestRendezvousDoneBeforeWait(c *gc.C) {
-	s.PatchValue(&meeting.Clock, s.clock)
+func TestRendezvousDoneBeforeWait(t *testing.T) {
+	c := qt.New(t)
```
Comment (mhilton) [inline meeting/meeting_test.go]: also defer c.Done() here.

File: meeting/meeting_test.go (hunk 2 above)
Comment (frankban) [inline meeting/meeting_test.go]: Should we use `c.Assert(err, qt.Equals, nil)` instead? (here and below)

File: meeting/meeting_test.go [hunk 4]
```diff
@@ -49,38 +35,40 @@ type nilMetrics struct{}
 func (nilMetrics) RequestCompleted(startTime time.Time) {}
 func (nilMetrics) RequestsExpired(count int)            {}
 
-func (s *suite) TestRendezvousWaitBeforeDone(c *gc.C) {
-	s.PatchValue(&meeting.Clock, s.clock)
+func TestRendezvousWaitBeforeDone(t *testing.T) {
+	c := qt.New(t)
+	clock := testclock.NewClock(epoch)
```
Comment (frankban) [inline meeting/meeting_test.go]: Defer c.Done() as we are patching variables.

File: meeting/meeting_test.go [hunk 5]
```diff
@@ -89,99 +77,103 @@ func (s *suite) TestRendezvousWaitBeforeDone(c *gc.C) {
...
 
-	c.Assert(atomic.LoadInt32(&count), gc.Equals, int32(0))
+	c.Assert(atomic.LoadInt32(&count), qt.Equals, int32(0))
 }
 
-func (s *suite) TestRendezvousDoneBeforeWait(c *gc.C) {
-	s.PatchValue(&meeting.Clock, s.clock)
+func TestRendezvousDoneBeforeWait(t *testing.T) {
+	c := qt.New(t)
+	clock := testclock.NewClock(epoch)
```
Comment (frankban) [inline meeting/meeting_test.go]: Defer c.Done() as we are patching variables.

Comment (frankban) [reply meeting/meeting_test.go]: Also in the tests below.

File: meeting/meeting_test.go (hunk 1 above)
Comment (rogpeppe) [reply meeting/meeting_test.go]: done

File: meeting/meeting_test.go (hunk 2 above)
Comment (rogpeppe) [reply meeting/meeting_test.go]: done

File: meeting/meeting_test.go (hunk 3 above)
Comment (rogpeppe) [reply meeting/meeting_test.go]: done

**Pull request merged**